vlib/orm/README.md

```diff
@@ -47,23 +60,55 @@ struct Child {
 }
 ```
 
-### Create
+To use the ORM, there is a special interface that lets you use the structs and V itself in queries.
```

Comment:
```suggestion
To use the ORM, there is a special expression that lets you use the structs and V itself in queries.
```
The word `interface` has a special meaning in V, and imho can be a bit confusing for the readers.